templates/frontend/pages/indexJournal.tpl

```diff
@@ -24,7 +24,7 @@
 
 	{if $homepageImage}
 		<div class="homepage_image">
-			<img src="{$publicFilesDir}/{$homepageImage.uploadName|escape:"url"}" alt="{$homepageImageAltText|escape}">
+			<img src="{$publicFilesDir}/{$homepageImage.uploadName|escape:"url"}" alt="{$homepageImageAltText|default:'null'}">
```

Comment:
I notice that even if I put alt text in with the homepage image, it doesn't appear. In retrospect, I think that `{$homepageImageAltText}` is not the right variable. Instead, the alt text is included in `{$homepageImage}`, similar to how `{$homepageImage.uploadName}` cointains the URL.

You can use the following to print the whole object to the template:

```
{$homepageImage|@print_r}
```

That is like doing `console.log(homepageImage)` in JS to inspect the object, but it will print directly to the template (not in the console).

Use that to find the property where the alt text is stored and use that to print it.